eventgrid/data-plane/src/test/java/com/microsoft/azure/eventgrid/customization/CustomizationTests.java

```diff
@@ -34,19 +43,36 @@
 public class CustomizationTests {
 
     @Test
-    public void consumeStorageBlobDeletedEventWithExtraProperty()throws IOException {
+    public void consumeStorageBlobDeletedEventWithExtraProperty() throws IOException {
         String jsonData = getTestPayloadFromFile("StorageBlobDeletedEventWithExtraProperty.json");
         //
         EventGridSubscriber eventGridSubscriber = new EventGridSubscriber();
 
-        EventGridEvent[] events = eventGridSubscriber.DeserializeEventGridEvents(jsonData);
+        EventGridEvent[] events = eventGridSubscriber.deserializeEventGridEvents(jsonData);
 
         Assert.assertNotNull(events);
         Assert.assertTrue(events[0].data() instanceof StorageBlobDeletedEventData);
         StorageBlobDeletedEventData eventData = (StorageBlobDeletedEventData)events[0].data();
         Assert.assertEquals("https://example.blob.core.windows.net/testcontainer/testfile.txt", eventData.url());
     }
 
+    @Test
+    public void ConsumeCustomEvents() throws IOException {
+        String jsonData = getTestPayloadFromFile("CustomEvents.json");
+        //
+        EventGridSubscriber eventGridSubscriber = new EventGridSubscriber();
+        eventGridSubscriber.putCustomEventMapping("Contoso.Items.ItemReceived", ContosoItemSentEventData.class);
+        eventGridSubscriber.putCustomEventMapping("Contoso.Items.ItemReceived", ContosoItemReceivedEventData.class);
+
```

Comment:
might be good to add a test for the new getallcustommappings() method .

Reply:
definitely, will add test coverage for operations on custom mapping.